syntax = "proto3";

package tags;

service Tags {
  rpc GetTags (ImageRequest) returns (TagsReply) {}
}

message ImageRequest {
  bytes file = 1;
}

message TagsReply {
  repeated string tags = 1;
}
